controllers/api/index.js

```diff
@@ -1,106 +1,98 @@
 'use strict';
 const express = require('express');
+const Joi = require('joi');
 
 const router = express.Router();
 
-const { pick } = require('lodash');
-const { body, validationResult } = require('express-validator/check');
-const { matchedData } = require('express-validator/filter');
-
-const { purifyUserInput } = require('../../modules/validators');
-const surveyService = require('../../services/surveys');
 const feedbackService = require('../../services/feedback');
+const surveyService = require('../../services/surveys');
 
 /**
  * API: Feedback endpoint
  */
-router.post(
-    '/feedback',
-    [
-        body('description')
-            .exists()
-            .not()
-            .isEmpty(),
-        body('message')
-            .exists()
-            .not()
-            .isEmpty()
-    ],
-    (req, res) => {
-        const formErrors = validationResult(req);
-        const formData = matchedData(req);
-        const messageSuccess = req.i18n.__('global.feedback.success');
-        const messageError = req.i18n.__('global.feedback.error');
+router.post('/feedback', async (req, res) => {
+    const schema = Joi.object({
+        description: Joi.string().required(),
+        message: Joi.string().required()
+    });
+
+    const validationResult = schema.validate(req.body, {
+        abortEarly: false,
+        stripUnknown: true
+    });
+
+    const messageSuccess = req.i18n.__('global.feedback.success');
+    const messageError = req.i18n.__('global.feedback.error');
+
+    if (validationResult.error) {
+        res.status(400).json({
+            status: 'error',
+            message: messageError,
+            err: validationResult.error.message
+        });
+    } else {
+        try {
+            const result = feedbackService.storeFeedback({
```

Comment:
Should this be `await feedbackService...`?

Reply:
Ack, good catch. I've updated this and also added some integration tests for both API endpoints to assert the value of the result in the response.